### PR TITLE
In this React final capstone project

### PR DESCRIPTION
**Pull request by Dagic-zewdu**
- [ ] Check this [video](https://www.loom.com/share/9b4a020efa1d47b49d66fe7b16d4b6d0) that shows the project at a glance

**API**
I choose [Narrativa API](https://covid19tracking.narrativa.com/index_en.html): COVID-19 data. API that show the stastistics of covid-19 at the Global and country level

**Design**
- I followed these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)):

**Interactions**
- Home page
  - When the page loads, the web app shows the list of countries with covid data.
  - Along with the country name, I displayed some total cases found in each country.
  - When the user clicks (or taps) on a country item, the application navigates to the details page.

- Details page
  - In the details page, the web app retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

**Testing requirements**
- Created unit tests for pure functions in your Redux code.
- Created integration tests for your application using the React Testing Library.
  - Mocked the access to the API so that your tests don't send actual requests.
  - Mocked the connection to the Redux Store.

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-nested-ternary */
```
Comment (Cathella) [inline src/components/Home/Home.js:1]: - [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.

File: src/components/Home/countryItem.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
```
Comment (Cathella) [inline src/components/Home/countryItem.js:1]: - [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.

File: src/components/Layouts/BasicLayout.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable react/prop-types */
```
Comment (Cathella) [inline src/components/Layouts/BasicLayout.js:2]: - [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.

File: src/components/Layouts/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types */
```
Comment (Cathella) [inline src/components/Layouts/Navbar.js:1]: - [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.

File: src/utils/testHelpers.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Cathella) [inline src/utils/testHelpers.js:1]: - [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @Dagic-zewdu 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Very nice video presentation.
✅ It has the required page structure and navigation
✅ The project uses Redux for state management 
✅ Data retrieved from the API are stored in the Redux store correctly.
✅ The project works as intended.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/utils/testHelpers.js (hunk 5 above)
Comment (rloterh) [inline src/utils/testHelpers.js:1]: - [ ] You have **disabled linter check** on your code. Please enable linter check on your code, as requested in the previous code review to ensure that your code is robust code and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Dagi, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍
- Descriptive Pull Request
- Good commit messages
- Redux state management properly implemented  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Please take your time to carefully improve aspects outlined in the previous code review(s), and please check to see that the changes are working as intended._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked **[OPTIONAL]** under this review_

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
Hi @Dagic-zewdu, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**